src/Sarif.Multitool/Rules/SARIF2008.ProvideSchema.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Json.Pointer;
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.CodeAnalysis.Sarif.Multitool.Rules
+{
+    public class ProvideSchema : SarifValidationSkimmerBase
+    {
+        public override MultiformatMessageString FullDescription => new MultiformatMessageString
+        {
+            Text = RuleResources.SARIF2008_ProvideSchema_FullDescription_Text
+        };
+
+        public override FailureLevel DefaultLevel => FailureLevel.Error;
```

Comment:
>Error [](start = 66, length = 5)

Warning (so rename resource strings accordingly).

Reply:
`run.$schema` is optional, so not error-level.

---
In reply to: [443047241](https://github.com/microsoft/sarif-sdk/pull/1921#discussion_r443047241) [](ancestors = 443047241)

Reply:
Oh, I see that the resources _do_ say "warning".

---
In reply to: [443047241](https://github.com/microsoft/sarif-sdk/pull/1921#discussion_r443047241) [](ancestors = 443047241)

Reply:
Nice catch!!!

---
In reply to: [443047522](https://github.com/microsoft/sarif-sdk/pull/1921#discussion_r443047522) [](ancestors = 443047522,443047241)